### PR TITLE
Modal branch PR

### PR DESCRIPTION
**Pull request by mertkantaroglu**
Hello,

I and my partner created this project. In this project, I created popup windows for cards. Cards popup by clicking the 'See Project' buttons. In addition, I also created an overlay to make it more visible to the viewer.

Thanks in advance.

### CODE REVIEW

File: menu.js [hunk 1]
```diff
@@ -1,19 +1,152 @@
...
+    image: 'Images/Snapshoot Portfolio (1).png',
+    technologies: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    link: 'link_to_live_version.com',
+    content1: 'Lorem ipsum it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    content2: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus repellendus mollitia saepe maxime vitae nostrum repellat eligendi amet voluptas modi repudiandae necessitatibus animi harum obcaecati at omnis laboriosam sapiente eum rem velit molestias',
+  },
+];
+
```
Comment (Mihndim2020) [inline menu.js:72]: - [x] Please make sure the information is visible of the pop ups, the images appear blurred and all the text does not appear, please see image below and kindly fix that. Please make sure all the information is displayed on the pop up. 
reviews1

review2

Please make sure all the information appears on the pop ups and it should be responsive.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @mertkantaroglu, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -1241,7 +1240,14 @@ button.project-button:active {
...
-    margin-top: 6%;
+    margin-top: 7%;
+    margin-left: 4%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
   }
```
Comment (Gopxfs) [inline styles.css]: You did a great job fixing the mobile popup 😄:
image

Please keep in mind that the desktop one still needs to be fixed, as requested by the previous reviewer 😉. Kindly make sure to adjust all the elements inside the container:
image

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @mertkantaroglu 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (mertkantaroglu) [reply styles.css]: Hi Gabriel. 
I don't know how you look at it but that doesn't make sense. When I check the project according to Figma dimensions it looks ok for both mobile and desktop. You can see the screenshots below.
Screen Shot 2023-01-06 at 22 20 44
Screen Shot 2023-01-06 at 22 20 23

**Review by nicupop729 — COMMENTED** [review]
Hello Team 👋,

_This is an additional review as a result of the second opinion request._

Reviewing your project I noticed the followings:

- you missed a very important project requirement, which is to render dynamically, with the use of JS, the work cards on the UI; at the moment, they are still hardcoded in the HTML file; because it was missed by the previous reviewers (@Mihndim2020  and @Gopxfs) to ask it from you, I will just mention it for you as OPTIONAL, but highly recommend to do so - it will make your life easier in the next module;
- also, your approach to building the pop is not the best; at the moment, many parts of your modal are hard-coded in the HTML file; take for example the technologies: even if on the work cards you display different technologies, in the pop-up you always display those hard-coded; you don't take full advantage of the JS and the dynamically rendering; I will leave this also as OPTIONAL because again, the previous reviewers, @Mihndim2020 and @Gopxfs missed to ask it from you;

Now going to your complaint: on the mobile view at the exact screen size of 375 X 820 px and on the desktop view at, again, the exact screen size of 1440 X 1200, you display properly the pop-up module. That's true.
But unfortunately, the users are not having all of them only these 2 screen sizes; actually, is very hard to find desktop screen sizes at 1440 X 1200. Most users have screen sizes bigger than this one.

That's why here at Microverse we are learning about Flexbox and Grid: to be able to build responsive elements, that will fit properly on multiple screen sizes.

If you don't refactor your work in order for the pop-up to fit multiple screen sizes, unfortunately, it means that you are not ready to go to the next module and that the lessons until now were not properly learned. 

Kindly, take your time, retake the necessary lesson from the past and try your best to fix your pop-up to fit different screen sizes.

There is no problem if you will repeat the week, is all to your advantage. Try to think in perspective.

When ready, ask for another review in the dashboard.

Keep rocking 🚀!

**Comment by mertkantaroglu** [comment]
Hello,

Per request, I made the page as responsive as I can. Please see attached screenshots for different screen sizes. I am aware that the transitions are not perfect but for various breakpoints, the page looks okay. Honestly, besides that, I don't know what else I can improve for the design. With my current knowledge that's the best, I can do.

Screen Shot 2023-01-07 at 11 06 08
Screen Shot 2023-01-07 at 11 00 53
Screen Shot 2023-01-07 at 11 12 11
Screen Shot 2023-01-07 at 11 07 08
Screen Shot 2023-01-07 at 11 08 07

File: index.html [hunk 3]
```diff
@@ -209,7 +259,7 @@ <h2 class="project-title">Uber Navigation</h2>
               <li><h3 class="css">css</h3></li>
               <li><h3 class="js">javaScript</h3></li>
             </ul>
-            <button class="project-button" type="button">See Project</button>
+            <button class="project-button project-button4" type="button">See Project</button>
           </div>
           </div>
         </div>
```
Comment (binhussen) [inline index.html:265]: [Optional] Kindly, you missed a very important project requirement, which is to render dynamically, with the use of JS, the work cards on the UI; at the moment, they are still hardcoded in the HTML file.👌👍 Please update it

File: menu.js [hunk 4]
```diff
@@ -1,19 +1,160 @@
...
+closeIcon.addEventListener('click', closeMobileMenu);
+
+// Popup Cards //
+const modal = document.querySelector('.popup-window');
+const overlay = document.querySelector('.overlay');
+const openModalBtn1 = document.querySelector('.project-button1');
+const openModalBtn2 = document.querySelector('.project-button2');
+const openModalBtn3 = document.querySelector('.project-button3');
+const openModalBtn4 = document.querySelector('.project-button4');
+const closeModalBtn = document.querySelector('.closeIcon-popup');
```
Comment (binhussen) [inline menu.js:29]: - [Optional]Kindly, if you were implemented the cards dynamically you will implement them in one button, so please implement the modal to be render dynamically and implemennt this one with `for each` 👌👌

**Review by binhussen — APPROVED** [review]
Hi @mertkantaroglu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**